python/tk_desktop/setup_project.py

```diff
@@ -60,12 +68,59 @@ def do_setup(self):
                                        "To re-setup a project, in a terminal window type: tank setup_project --force\n\n"
                                        "Alternatively, you can go into shotgun and clear the Project.tank_name field\n"
                                        "and delete all pipeline configurations for your project.")
-            ret = error_dialog.exec_()
+            error_dialog.exec_()
+
+        except TankUserPermissionsError, e:
+            error_dialog = ErrorDialog("Toolkit Setup Error",
+                                       "You do not have sufficient permissions in Shotgun to setup Toolkit for "
+                                       "project '%s'.\n\nContact a site administrator for assistance." %
+                                        self.project["name"]
+            )
+            error_dialog.exec_()
 
         finally:
             if is_on_top: 
                 self._set_top_window_on_top(True)
 
+    def show_help_popup(self):
+        """
+        Display a help screen
+        """
+        # Not sure what to put here.
+        pass
```

Comment:
Any ideas for what should be implemented in this spot?  The launcher help popup looks really great.  Should we try to emulate something like that here? Or is a nicely formatted link to the existing docs sufficient?